native/specter_nif/src/track.rs

```diff
@@ -0,0 +1,69 @@
+use crate::state::Ref;
+use crate::{atoms, task};
+use rustler::{Encoder, Env, ResourceArc, Term};
+use std::fs::File;
+use std::io::BufReader;
+use tokio::time::Duration;
+use webrtc::media::io::h264_reader::H264Reader;
+use webrtc::media::Sample;
+
+#[rustler::nif]
+pub fn play_from_file<'a>(
+    env: Env<'a>,
+    resource: ResourceArc<Ref>,
+    track_uuid: Term<'a>,
+    path: Term<'a>,
+) -> Term<'a> {
+    let mut state = match resource.0.lock() {
+        Err(_) => return (atoms::error(), atoms::lock_fail()).encode(env),
+        Ok(guard) => guard,
+    };
+
+    let decoded_track_uuid: String = track_uuid.decode().unwrap();
+    let track = state
+        .get_track_local_static_sample(&decoded_track_uuid)
+        .unwrap()
+        .clone();
+
+    let pid = state.pid.clone();
+    let mut msg_env = rustler::env::OwnedEnv::new();
+    let decoded_path: String = path.decode().unwrap();
+
+    task::spawn(async move {
+        // Open a H264 file and start reading using our H264Reader
+        let file = File::open(&decoded_path).unwrap();
+        let reader = BufReader::new(file);
+        let mut h264 = H264Reader::new(reader);
+
+        println!("play video from disk file {}", decoded_path);
```

Comment:
In C NIFs I saw the pattern (maybe in code that you wrote?) of being able to compile with `DEBUG` to conditionally compile these in. I haven't looked at how to do this in Rust/rustler, but wondered what you thought about it. It seems like these would be useful when debugging?

Reply:
I see this: https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/log.html. Maybe it would be helpful?

Reply:
I add Rust logger and when you run it using `mix run` then it looks like this 
![image](https://user-images.githubusercontent.com/24651552/162063241-79203145-ad1b-4fa5-9526-8c6d4647d188.png)


However, when I try to run it from iex then I get
![image](https://user-images.githubusercontent.com/24651552/162063366-a96622f2-467a-4520-bcb1-bd81d90b9426.png)


And I am not sure why and how to fix it :thinking: 
Looks like the problem is related with using `iex`.

Reply:
Hmm... I think I've seen this before in C NIFs and iex... Just randomly, if you prepend `\r` to the debug statement does it jump back to the beginning of the line?

Reply:
Yup, that helps. Should I put it? I feel like we will end up having `\r` in some places and in some places it will be missing

Reply:
We can always extract a helper if it seems like it becomes a problem.

I think the problem for me would be to remember to test in different scenarios to verify that I do the correct thing, so maybe it makes sense to add a helper now.